README.md

```diff
@@ -103,6 +106,41 @@ sensu_asset { 'test':
 }
 ```
 
+### Updating this module from 4.x to 5.x
+
+This module begins supporting Sensu Go 6 with version >= 5.0.0
+
+**NOTE** Upgrading to support Sensu Go 6 requires backends have Puppet applied before agents will begin to work as there is an agent specifc Sensu user and role added to support modifying agent entities via the API.
+
+#### Changes for backend
+
+There is a manual step to perform to upgrade the sensu-backend after upgrading the backend to 6.x.
+This module provides the `sensu::backend_upgrade` bolt task as a way to execute the necessary `sensu-backend upgrade` command.
+
+#### Changes for agents
+
+Beginning with Sensu Go 6, some changes to `agent.yml` will only bootstrap an agent entity, they will not update the entity.
+If you wish to make changes to values such as `subscriptions`, `labels` or `annotations` after a host is added to Sensu this must be done
+via the Sensu Go API. To support this it's now required that agents have the ability to make API calls.
+
+In order to ensure agents can make API calls either via API or sensuctl the agent must be told about the admin password:
+
+```
+class { 'sensu':
+  agent_entity_config_password => 'supersecret',
+}
+class { 'sensu::agent':
+  ...
+}
+```
+
+See [API Providers](#api-providers) for example Hiera that can be used in a file like `common.yaml` to easily share the admin password with agents.
+
+This module will still continue to write subscriptions and other agent configurations to `agent.yml` so that if an agent entity is deleted it can be recreated
+by restarting the `sensu-agent` service.
+
+**NOTE**: At this time redaction of labels or annotations is not supported and will cause this module to produce errors. Do not use label or annotation redaction at this time. See [limitations](#limitations) for details.
```

Comment:
Remove this phrase as it is redundant. 

redaction. <strike>at this time</strike>

Reply:
Phrase removed